server_test.go

```diff
@@ -393,7 +394,7 @@ func Test_MultiplePost(t *testing.T) {
 			Resource:  "test",
 			DataType:  event3.NOTIFICATION,
 			ValueType: event3.ENUMERATION,
-			Value:     event3.ACQUIRING_SYNC,
+			Value:     ptp.ACQUIRING_SYNC,
```

Comment:
the type is not using the defined types, can you update that in the test .

Reply:
updated to use `ptp.PtpStateChange`